go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -1612,11 +1612,21 @@ func (s *VtctldServer) UpdateThrottlerConfig(ctx context.Context, req *vtctldata
 	}
 	defer unlock(&err)
 
-	_, err = s.ts.UpdateSrvKeyspaceThrottlerConfig(ctx, req.Keyspace, []string{}, update)
+	ki, err := s.ts.GetKeyspace(ctx, req.Keyspace)
 	if err != nil {
 		return nil, err
 	}
-	return &vtctldatapb.UpdateThrottlerConfigResponse{}, nil
+
+	ki.ThrottlerConfig = update(ki.ThrottlerConfig)
+
+	err = s.ts.UpdateKeyspace(ctx, ki)
```

Comment:
I think it doesn't matter, or shall I say "it's OK", but just noting that `UpdateThrottlerConfig` executed on different `SrvKeyspace`s will override each other's config.

Reply:
1) this is fine
2) each `UpdateSrvKeyspaceThrottlerConfig` below actually updates all cells